plugins/docker/docker_

```diff
@@ -186,8 +243,10 @@ def main():
             print("graph_vlabel CPU usage (%)")
             print("graph_category virtualization")
             print("graph_info This graph shows docker container CPU usage.")
-            for container in client.containers.list():
+            print("graph_total Total CPU usage")
+            for container in client.containers.list(all=True):
```

Comment:
Maybe `sorted` would be helpful here, too. But maybe the `list` method already produces a stable order - I do not know.